### ISSUE TITLE
enhancement: add etymology info in config.php to map

### ISSUE THREAD

**Issue by ynux** [EqualStreetNames/equalstreetnames#145]
For streets (or more precisely osm ids) in the config.php, the gender is shown as the appropriate color in the map. It would be nice if etymology data could also be shown at mouseclick on the street, like "named after some utterly boring local politician that neither Wikipedia nor Wikidata can be bothered with". 
So far, some comments were added. The info could be in a third comma separated field.

**Comment by jbelien** [comment]
I gave some thoughts about this and actually, I'm not sure how I feel about it.

The goal of the project is to be fully based on data from OpenStreetMap and Wikidata.
I think that if we have information about a person those data should go into Wikidata and linked to OpenStreetMap using `name:etymology:wikidata` tag.
The data input from `config.php` should only be used when we don't have enough information to create a Wikidata item (for instance, gender deduced from the firstname but no idea how that person is).

**Comment by jbelien** [comment]
I would be interested to have your insight on this topic @RobinLinde @gislars @stalker314314 ?

**Comment by RobinLinde** [comment]
TL;DR: I'm for adding to Wikidata, even if it's only very basic

Personally I'm for adding everything to Wikidata, since it allows anyone to add more data and context to a person, but sometimes adding just some basic details can take a lot of time. For example, [this](https://www.wikidata.org/wiki/Q88359868) took me something like half an hour to do. Might be interesting for once, but if you have to do this for more streets, it quickly gets annoying and tedious. Though I guess adding staying very basic, with name, gender and a small description can be enough.

Personally I also take time to add `wikidata` tags, and add `P138` on Wikidata to show etymology. This also allows making [this kind](https://www.wikidata.org/wiki/User:Robin2001/Streets_in_Assen) of pages.

Since I'm already typing for so long, just another side-note: I think that creating a dedicated editor might simplify the process of gathering data.

**Comment by ynux** [comment]
What is this project about? We fight against too much public space given to CIS men. And now we add hundreds of them to Wikidata? That doesn't feel right.

**Comment by jbelien** [comment]
> What is this project about?

This project is about creating a map (and statistics) to highlight the inequality between genders in the naming of the streets of a city (and to suggest alternatives or suggestions for new streetnames, at least for Brussels where the project was initiated - see "Suggestions" in <https://equalstreetnames.brussels/>).

This project is also about showing the power of open-data !
That's why it has been decided to use _OpenStreetMap_ and _Wikidata_.
I think that adding data limited to this project (by using the `config.php` file) do not fit that open-data purpose.

That being said, you can decide to add WIkipedia pages (and/or Wikidata items) only for the non cis-gender men in Munich (and use the `config.php` file for the cis-gender men).
@Manonbd (former employee of Open Knowledge Belgium and initiator of this EqualStreetNames project) organizes regular workshops focused on adding Wikipedia pages (and Wikidata items) for women, trans-gender men, and non-binary people.

**Comment by gislars** [comment]
My opinion is EqualStreetNames is a great visualization for the gender inequality  in our street networks. As I was setting it up for Berlin I was thinking about to extend it to visualize more Wikidata categories. It might also be interesting how much streets are named after other cities, professions, plants... but I think it is out of scope or it would be an other project. 

@ynux I understand your point. I don't think not adding more CIS men to Wikidata would solve the inequality around us. Wikidata is gathering facts and with EqualStreetNames + OSM these are visualized and everyone can draw their own conclusions.

**Comment by eMerzh** [comment]
adding a few cents here, as I recently did a new city... and indeed, there are a few streets that are named after an old politician that no one knows about, and of course Wikipedia page was rejected and not sure about wikidata but should probably be the same kind of answer ... 

those ends up beeing in the config, but then there isn't any info attached...  not sure we want to bother too much with this but data not in Wikidata is a real thing

**Comment by ynux** [comment]
I guess the situation depends on your city. We did all the streets in our district of Munich. Here we have 169 streets, of which 122 are named after a person. Of those, 52 do not have Wikidata entries (50 men, 2 women). 
Maybe our district is exceedingly boring, streetnamewise. It rapidly grew around 1900, 27 of those men were taken as hostages in 1632, and that is all that they are known for. Those streets all got their names in 1901. 

Of course the elegance of the project is in using Wikidata. It is linked data at its best. 
But then reality rises its ugly head, with names like:
```
M,"Schifferlstraße: Schifferl, eine der 42 Münchner Geiseln König Gustavs II. Adolf"
M,"Schrottstraße: benannt nach dem Magistratsrat Ludwig Schrott"
M,"Siglstraße: Sigl, eine der 42 Münchner Geiseln König Gustavs II. Adolf"
M,"Stroblstraße: Strobl, eine der 42 Münchner Geiseln König Gustavs II. Adolf"
M,"Sturmstraße: Albin Sturm, Lehrer"
M,"Stöberlstraße: Stöberl, eine der 42 Münchner Geiseln König Gustavs II. Adolf"
M,"Stögerstraße: nach dem Buchhändler und Hofbibliothekar Josef Stöger"
```
and i really wouldn't enjoy entering all of them into Wikidata. Munich has over 6000 streets, if this is representative, this would mean adding 2500 entries of low relevance to Wikipedia.

**Comment by ynux** [comment]
To satisfy some local curiosity we created a variant that displays "local data" living in a csv of this form:
```
id,street_name,person_name,gender,description,language,year_named,year_built
4271290,Agricolaplatz,Agricola,M,"eine der 42 Münchner Geiseln König Gustavs II. Adolf",de,1901,1901
4218701,Agricolastraße,Agricola,M,"eine der 42 Münchner Geiseln König Gustavs II. Adolf",de,1901,1901
4271289,Agricolastraße,Agricola,M,"eine der 42 Münchner Geiseln König Gustavs II. Adolf",de,1901,1901
69688189,Agricolastraße,Agricola,M,"eine der 42 Münchner Geiseln König Gustavs II. Adolf",de,1901,1901
```
The `year_named,year_built`is for some future experiments. For several languages, the data would probably need restructuring.
The map is created from special branches: [super module](https://github.com/ynux/equalstreetnames/tree/laim) and [munich sub module](https://github.com/EqualStreetNames/equalstreetnames-munich/tree/laim)
and it is presently displayed at
[https://ynux.github.io/ ](https://ynux.github.io/ )

**Comment by jbelien** [comment]
Interesting ! I l like that separate CSV file logic. 👍
